libs/design-system/src/panda/sizes.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * Represents the size of an element.
+ *
+ * Used for properties like width and height.
+ */
+export const SIZES_TOKENS = {
+  '25': {
+    value: '0.25rem',
+    type: 'sizes',
+  },
+  '50': {
+    value: '0.5rem',
+    type: 'sizes',
+  },
+  '75': {
+    value: '0.75rem',
+    type: 'sizes',
+  },
+  '100': {
+    value: '1rem',
+    type: 'sizes',
+  },
+  '125': {
+    value: '1.25rem',
+    type: 'sizes',
+  },
+  '150': {
+    value: '1.5rem',
+    type: 'sizes',
+  },
```

Comment:
Good question! Consider these in pre-release since they're not derived from Supernova yet, and I took them directly from [Notion](https://www.notion.so/novuhq/Design-Language-and-Token-specifications-Eugene-c1bda2e788ff4993a4ff579ec40e6946) (which is missing 175). I will add for now just in case